drivers/claims_reporting/app/models/claims_reporting/medical_claim.rb

```diff
@@ -9,6 +9,9 @@ module ClaimsReporting
   class MedicalClaim < HealthBase
     extend Memoist
 
+    ANNUAL_WELLCARE_PROCEDURE_CODES = [(99381...99387).to_a, (99391...99397).to_a, 99461, 'G0438', 'G0439', 'T1015'].flatten.map(&:to_s).freeze
+    ANNUAL_WELLCARE_DIAGNOSIS_CODE_PREFIXES = ['Z00', 'Z02', 'Z76'].freeze
```

Comment:
@awisema do you think it's safe to include everything with these prefixes, or should it be more granular for Z02 and Z76?

Reply:
I think its fine, but if you are worried, there is enough data at this point, that if you want, we can run the SQL and see if there looks to be anything they don't include...